src/main/java/org/akaza/openclinica/control/managestudy/ListEventDefinitionServlet.java

```diff
@@ -207,15 +207,16 @@ private boolean isLockable(StudyEventDefinitionBean sed, StudyEventDAO sedao, Ev
         ArrayList events = (ArrayList) sedao.findAllByDefinition(sed.getId());
         for (int j = 0; j < events.size(); j++) {
             StudyEventBean event = (StudyEventBean) events.get(j);
-            if (!(event.getStatus().equals(Status.AVAILABLE) || event.getStatus().equals(Status.DELETED))) {
+
+                if ((event.getRemoved()!=null && event.getRemoved()) || (event.getArchived()!=null && event.getArchived())) {
```

Comment:
looks like there are a lot of these instances. Please update all these to use BooleanUtils